homeassistant/components/caldav/calendar.py

```diff
@@ -66,6 +68,7 @@ def setup_platform(hass, config, add_entities, disc_info=None):
     url = config[CONF_URL]
     username = config.get(CONF_USERNAME)
     password = config.get(CONF_PASSWORD)
+    period = config.get(CONF_PERIOD)
```

Comment:
Please use `dict[key]` for required config schema keys and keys with default schema values.

Reply:
```suggestion
    period = config[CONF_PERIOD]
```